recipes/zeromq/cppzmq_git.oe

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "C++ bindings for ZeroMQ"
+HOMEPAGE = "http://www.zeromq.org"
+LICENSE = "MIT"
+SRC_URI = "git://github.com/zeromq/cppzmq.git;protocol=http;${SRC_REV}"
+SRC_REV = "commit=a88bf3e0b0bc6ed5f5b25a58f8997a1dae374c8b"
+
+S = "${SRCDIR}/cppzmq"
+
+do_install () {
+    install -d ${D}${includedir}
+    install -m 0755 ${S}/zmq.hpp ${D}${includedir}
+}
+
+PACKAGES = "${PN}"
+FILES_${PN} = "${includedir}"
+DEPENDS_${PN} = "libzmq"
+RDEPENDS_${PN} = "libzmq-dev"
```

Comment:
I removed the comment, because I changed the packaging enough to completely invalidate the comment.

What specifically would you like to have a comment about?  The fact that DEPENDS_${PN} and RDEPENDS_${PN} is different?